services/server/src/wss/handlers/connection.ts

```diff
@@ -23,8 +23,8 @@ export class Connection extends WssHandler {
       const user = this.user;
       if (isGhost(user)) return;
 
-      const info = await users.update(user.id, { online: true });
-      this.announceStatus(info);
+      await cache.onlineStatus.addUser(user.id);
+      this.announceStatus(user.id, true);
```

Comment:
I believe we should update `announceStatus` to accept an object as a param, not to cause a confusion about what `true/false` means. 


```suggestion
      this.announceStatus({userId: user.id, online: true});
```